code/script.js

```diff
@@ -1,51 +1,312 @@
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 What kinds of stardust?👇</option>
+        <option value="Black Crystals">Black Crystals, hand-picked from black holes 😱</option>
+        <option value="Pink Unicorn Poo">Pink Unicorn Poo</option>
+        <option value="Lactose Free Star Dust">Regular Star Dust (Lactose Free)</option>
+      </select>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight;
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion(select.value))
 }
 
-// Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+const showPaymentOptions = (paymentoptions) => {
+  questionNumber++
+
+  botReply(`${paymentoptions} coming up! Do you want to pay with cash 💰, diamonds 💎, or vanilla ice cream 🍦?`)
+
+  inputWrapper.innerHTML = `
+    <button id="cash">💰</button>
+    <button id="diamonds">💎</button>
+    <button id="ice cream">🍦</button>
+  `
+
+  document
+    .getElementById('cash')
+    .addEventListener('click', () => nextQuestion('cash'))
+    document
+    .getElementById('diamonds')
+    .addEventListener('click', () => nextQuestion('diamonds'))    
+  document
+    .getElementById('ice cream')
+    .addEventListener('click', () => nextQuestion('ice cream'))
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000);
+
+
+const showPrice = (payment) => {
+  questionNumber++
+
+  let price
+  if (payment === 'cash') {
+    price = `${dollar_Id} 💰`;
+  } else if (payment === 'diamonds'){
+    price = `${diamond_Id} 💎`;
+  } else if (payment === 'ice cream'){
+    price = `${icecream_Id}`;
+  }
+
+  botReply(
+    `Ok then, the total price will be ${price}  Are you sure you want to order this?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="restart">NO</button>
+    <button id="confirm">YES</button>
+  `
+
+  document.getElementById('restart').addEventListener('click', () => {
+    location.reload()
+    return false
+  })
+  document
+    .getElementById('confirm')
+    .addEventListener('click', () => nextQuestion('Yes!'))
+}
+
+const thankYou = () => {
+  botReply(`OK! 👽 Your order delivery will arrive any minute now 🛸 Have a heavenly day!`)
+  inputWrapper.innerHTML = ``
+}
```

Comment:
I would've liked a ```
     location.reload()
     return false``` 
return button also if the user clicks yes (that they are sure they want to order this). Right now the chat only starts over if the user clicks no.